src/ui/styles/tabs/w1/_index.css

```
@@ -1,3 +1,4 @@
+@import url("./_companions.css");
```

Stylelint (17.13.0)

[error] Expected "url("./_companions.css")" to be ""./_companions.css"" (import-notation)

(import-notation)